.github/workflows/rust-tests.yml

```diff
@@ -84,14 +84,11 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - name: Install Rust
-      uses: dtolnay/rust-toolchain@stable
+      uses: actions-rs/toolchain@v1
```

Comment:
Please do comment why we're not using `dtolnay` here. Otherwise we'll end up changing it again sooner or later and breaking it.